### DISCUSSION TITLE
UI redesign idea

### DISCUSSION THREAD

**Discussion by CaptainKokomo** [adieyal/sd-dynamic-prompts#610] [Ideas]
don't know if this is already a thing. I did take a look but didn't find much. 

Would it be possible to make an addition to the UI?
I was thinking something along the lines of drop down menus for wildcards or categories of wildcards and a way to generate a random prompt using what was selected etc. 

Is this possible or is my adhd showing again?

**Comment by adieyal** [comment]
Could you elaborate? A concrete example would be helpful.

**Comment by CaptainKokomo** [comment]
I can try. I'm not a programmer. I imagined something simple but I guess nothing ever is.

I was thinking of having a smaller prompt box in a tab. With maybe drag and drop icons that represent a drop down menu below of either a wildcard category or a specific wildcard. Even a copy and paste thing at least.
Once you have it how you want it send it to txt2img or img2img. Could be a preview to see how the prompt will be written to prevent format errors.

**Comment by DrJKL** [comment]
I think I have something that gets part of the way there. Still needs a bit of work.

https://drjkl.github.io/prompt-crafter/
